README.md

```diff
@@ -61,9 +60,11 @@ In [main.rs](src/examples/string_from_move_print/main.rs), we define these RAPs
 ```rust
 /*--- BEGIN Variable Definitions ---
 Owner x; Owner y;
-Function String::from()
+Function String::from();
 --- END Variable Definitions ---*/
 ```
+Notice that the definition header now could be automatically generated by running the [view_examples.sh](rustviz_mdbook/view_examples.sh) once. If any incorrect information appeared at the generated header, you could manully edit it by refering to the following documentation.
```

Comment:
Remove "Notice that", change "could" -> "can" (present tense, not past, in things like this)